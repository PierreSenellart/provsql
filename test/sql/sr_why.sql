\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

CREATE TABLE result_why AS SELECT
  p1.city,
  sr_why(provenance(), 'personnel_name') AS why
FROM personnel p1, personnel p2
WHERE p1.city = p2.city AND p1.id < p2.id
GROUP BY p1.city
ORDER BY p1.city;

SELECT remove_provenance('result_why');
SELECT city, why
FROM result_why;

DROP TABLE result_why;
