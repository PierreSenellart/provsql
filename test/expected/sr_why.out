\set ECHO none
remove_provenance

(1 row)
city|why
Berlin|{{Ellen,Susan}}
New York|{{John,Paul}}
Paris|{{Dave,Magdalen},{Dave,Nancy},{Magdalen,Nancy}}
(3 rows)
